[PATCH] using the dynamic library

Upstream-Status:  Inappropriate [configuration]

building statically-linked sqlite3 failed since sqlite3.o is generated in 
different dir, even if link successes, the size of sqlite3 is become larger,
so use the dynamic link, ref: http://patchwork.openembedded.org/patch/93293/

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 88bc23d..fe50f20 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,7 +7,8 @@ libsqlite3_la_LDFLAGS = -no-undefined -version-info 8:6:8
 
 bin_PROGRAMS = sqlite3
 sqlite3_SOURCES = shell.c sqlite3.h
-sqlite3_LDADD = sqlite3.$(OBJEXT) @READLINE_LIBS@
+sqlite3_LDADD = @READLINE_LIBS@ libsqlite3.la
+
 
 include_HEADERS = sqlite3.h sqlite3ext.h
 
-- 
1.9.1

